src/docs/product/integrations/integration-platform/webhooks/comments.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Comments
+sidebar_order: 5
+description: "Learn more about Sentry's Integration Platform comment webhooks."
+---
+
+Sentry integrations that have subscribed to comment webhooks can receive information about a comment being created and changing state.
+
+## Sentry-Hook-Resource Header
+
+`'Sentry-Hook-Resource': 'comment'`
+
+## Attributes
+
+### action
+
+- type: string
```

Comment:
I'm tempted to take out the types here (in all the other pages too) since it's pretty clear by looking at the example payload what they are. If I do that I can simplify this and just list the attributes along with the description, but I am torn ...

Reply:
If someone's troubleshooting, having that information spelled out can be helpful. And having it spelled out is probably a bit more accessible than only having it implied by the code snippet.